tutorials/nbconverted/general_usage.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Pycytominer General Usage Walkthrough!
+# 
+# Welcome to this walkthrough where we will guide you through the process of extracting single cell morphology features using the [`pycytominer`](https://github.com/cytomining/pycytominer) API.
+# 
+# For this walkthrough, we will be working with the NF1-Schwann cell morphology dataset. 
+# If you would like the more information about this dataset, you can refer to this [repository](https://github.com/WayScience/Benchmarking_NF1_data)
+# 
+# From the mentioned repo, we specifically used this [dataset](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/4_processing_features/data/Plate2/CellProfiler) and the associated [metadata](https://github.com/WayScience/Benchmarking_NF1_data/tree/main/3_extracting_features/metadata) to generate the walkthrough. 
+# 
+# 
+# Let's get started with the walkthrough below!
+
+# In[17]:
+
+
+import pathlib
+
+import pandas as pd
+
+# pycytominer imports
+from pycytominer.cyto_utils.cells import SingleCells
+from pycytominer import annotate, normalize, feature_select
+
+# ignore warnings
+import warnings
+
+warnings.filterwarnings("ignore")
+
+
+# ## About the inputs
+# 
+
+# In this section, we will set up the expected input and output paths that will be generated throughout this walkthrough. Let's take a look at the explanation of these inputs and outputs.
+# 
+# For this workflow, we have two main inputs:
+# 
+# - **plate_data**: This contains the quantified single-cell morphology features that we'll be working with.
+# - **plate_map**: This contains additional information related to the cells, providing valuable context of our single-cell morphology dataset.
+# 
+# Now, let's explore the outputs generated in this workflow. In this walkthrough, we will be generating four profiles:
+# 
+# - **sc_profile_path***: This refers to the single-cell morphology profile that will be generated.
+# - **anno_profile_path**: This corresponds to the annotated single-cell morphology profile.
```

Comment:
```suggestion
# - **anno_profile_path**: This corresponds to the annotated single-cell morphology profile, meaning the information from the **plate_map** is added to each single-cell.
```
Just a recommendation to add what the annotation means/describing that you use the platemap file here.